src/main/java/org/reficio/p2/utils/JarUtils.java

```diff
@@ -96,9 +108,12 @@ public static void adjustFeatureQualifierVersionWithTimestamp(File inputFile, Fi
             if (jar != null) {
                 jar.close();
             }
+            if (null!=newXml) {
```

Comment:
I don't want to begin war between value != null and null!=value :) but I've never used the null-first approach in the code of this project.
I find it better when there's only one convention used in the code - for consistency and readability - for this project it's null-last. Could you adjust all if's that are null-first in your Pull-Request? 
It' just looks ugly otherwise -> look here there are two if statements one after the other and they use different conventions.